packages/components/heading/index.js

```diff
@@ -5,6 +5,7 @@ import { Box as ChakraBox, ThemeProvider, Box } from '@chakra-ui/core';
 import { space, typography, color } from 'helpers/styleProps';
 import getAllowedProps from 'helpers/getAllowedProps';
 import theme from 'helpers/theme';
+import { h1, h2 } from './theme';
 
 const Heading = ({ children, variant, override }) => (
   <ThemeProvider theme={theme}>
```

Comment:
This has to come from with theme right ?

Reply:
Nope that's the config file for each components. I named it as h1 and h2 and imported it.

Reply:
If you are saying about the Theme provider, we can use withTheme but cannot export dot notation components with HOC. Need to find a better way. So I fell back to straight forward approach.

Reply:
ok, we will find a better way. But for the final build using rollup the themeless component should be exported

Reply:
Sure. I will research on this today.

Reply:
Not urgent since we can do when we start building. Probably next week.

Reply:
Added to Trello